File: generators/entity/templates/_ModelMapper.java
package <%= groupCases.splitByDot %>.<%= nameCases.splitByDot %>.mapper;

import com.baomidou.mybatisplus.mapper.BaseMapper;
import <%= groupCases.splitByDot %>.<%= nameCases.splitByDot %>.model.<%= entityClass %>;

public interface <%= entityClass %>Mapper extends BaseMapper<<%= entityClass %>> {
}

File: generators/entity/templates/_ModelMapper.kt
package <%= groupCases.splitByDot %>.<%= nameCases.splitByDot %>.mapper

import com.baomidou.mybatisplus.mapper.BaseMapper
import <%= groupCases.splitByDot %>.<%= nameCases.splitByDot %>.model.<%= entityClass %>

interface <%= entityClass %>Mapper : BaseMapper<<%= entityClass %>> {
}
